api/turing/config/config.go

```diff
@@ -460,8 +466,6 @@ func setDefaultValues(v *viper.Viper) {
 
 	v.SetDefault("OpenapiConfig::ValidationEnabled", "true")
 	v.SetDefault("OpenapiConfig::SpecFile", "api/openapi.yaml")
-	v.SetDefault("OpenapiConfig::SwaggerUIConfig::ServingDirectory", "")
-	v.SetDefault("OpenapiConfig::SwaggerUIConfig::ServingPath", "/api-docs/")
```

Comment:
Why do we need to remove this?

Reply:
I'm wondering why was it set in the first place, on a local setup, `/api-docs/` is non existent, if it's used in the deployment then I have edited the _helpers.tpl to reflect this value. And also as issues mentioned here: https://github.com/gojek/turing/pull/140#discussion_r760959454

Reply:
I just remembered, because if these defaults are set, then it will never use the handwritten API specs when developing locally. Most of us will definitely forget to generate the bundled yaml config as per my logic of use bundled file if possible else use the handwritten api spec.

Reply:
> I'm wondering why was it set in the first place, on a local setup, /api-docs/ is non existent

`/api-docs/` is a _seerving path_ i.e. swagger-ui will be accessible under <host:port>/api-docs/

> if it's used in the deployment then I have edited the _helpers.tpl to reflect this value

Helm chart doesn't cover local development scenario (when the app runs from a local machine or via docker compose). The default value was set to keep it consistent with https://github.com/gojek/turing/blob/b603d60763fd73d7b0104d5e88d1402b15d0e468/api/turing/config/config.go#L459. If you want to standardize the way how the default values are set, then it should be consistent with other config values.

> I just remembered, because if these defaults are set, then it will never use the handwritten API specs when developing locally. 
We are talking about Swagger UI here, not about the spec file. So, the default value only sets under which path Swagger UI should be served (`ServingDirectory` is still empty, so by default there is noting to serve)

Reply:
Ok the other alternative is to stat the file if it exists then serve, if it does not exist we give a warning

Reply:
Actually I'll use your `spaCfg := cfg.OpenapiConfig.SwaggerUIConfig; spaCfg != nil && len(spaCfg.ServingDirectory) > 0` logic.